components/calendar/src/iso.rs

```diff
@@ -458,9 +467,21 @@ impl Iso {
     // Lisp code reference: https://github.com/EdReingold/calendar-code2/blob/1ee51ecfaae6f856b0d7de3e36e9042100b4f424/calendar.l#L1237-L1258
     pub(crate) fn iso_from_fixed(date: i32) -> Date<Iso> {
         let year = Self::iso_year_from_fixed(date);
-        let prior_days = date - Self::iso_new_year(year);
+        // Increase the date by 365 (see below why) and set it to next_year, only if the current year is MIN_YEAR
+        let (fixed_date, adjusted_year) = if year == MIN_YEAR {
```

Comment:
suggestion: this deviates from the lisp code; can we factor this out into a separate appropriately-named function and call it with a comment mentioning that it is a deviation (and why)